docs/MODULES.md

```diff
@@ -303,18 +303,56 @@ or with IPv6 and subnet
 
 ##### IPTunnel Client and Internet Peer
 
-If you wish to run an IPTunnel client on the same node as as an Internet peer, you will need to create a route to that node over the Internet connection available.
+If you wish to run an IPTunnel client on the same node as as an Internet peer, you will need to use a method of rerouting the path to the Yggdrasill peers Internet connection available instead over yggdrasil.
 
-This must be done because since all Internet traffic will be redirected over the tunnel and a route is not created to the peer, Yggdrasil peering will try to route over the tunnel. Since the tunnel depends on the peer, the peer will not function, which will collapse the tunnel. In other words, the peer tries to feed packets over the tunnel, that tries to feed packets to the peer, causing both to fail.
+This must be done because since all Internet traffic will be redirected over the tunnel.  If a route is not created to the peer, Yggdrasil peering will try to route over the tunnel. Since the tunnel depends on the peer, the peer will not function, which will collapse the tunnel. In other words, the peer tries to feed packets over the tunnel, that tries to feed packets to the peer, causing both to fail.
```

Comment:
```suggestion
This must be done because all Internet traffic will be redirected over the tunnel. If a route is not created to the peer, Yggdrasil peering will try to route over the tunnel. Since the tunnel depends on the peer, the peer will not function, which will collapse the tunnel. In other words, the peer tries to feed packets over the tunnel, that tries to feed packets to the peer, causing both to fail.
```